refine

```diff
@@ -942,7 +942,7 @@ elif [ "$OS" = "linux" ] ; then
 fi
 
 echo You have "$freeRam"M of free memory. 
-echo Your current configuration will allow to use $REFINE_MEMORY of memory.
+echo Your current configuration is set to use $REFINE_MEMORY of memory.
```

Comment:
This PR contains two unrelated changes… It's generally better to use separate branches and PRs for that.

Reply:
@wetneb We know.  Jacky is just fixing some incidentals that I bring up in our hangouts.  There just minor typo fixes. I know, I know, it makes the changes look dirty. I know.  But for typos, I am OK with dirty changes of typos within feature pulls.  I know your probably not, but I and Jacky are OK with them.  We're moving fast, because we have an agenda to get this release done, moved out...so that we can actually begin to plan better and focus on "the 1st quarter fun".